CHANGELOG.md

```diff
@@ -3,7 +3,12 @@
 ## 2.2.0
 
 - Added option to specify the preferred dafny version for automatic installation
+- Empty compiler/language server paths are now treated as unconfigured (use automatic installation)
+- Now accepting .NET 6.0 as a supported platform
```

Comment:
Nitpick I'd phrase this and similar messages as `Accept .NET 6.0 as a supported platform`, so you don't have to write every change in past tense.